arches/app/src/components/ControlledListManager/AddMetadata.vue

```diff
@@ -0,0 +1,94 @@
+<script setup lang="ts">
+import arches from "arches";
+import { computed, inject } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+
+import { ARCHES_CHROME_BLUE } from "@/theme.ts";
+import { itemKey } from "@/components/ControlledListManager/constants.ts";
+
+import type { Ref } from "vue";
+import type {
+    ControlledListItem,
+    ControlledListItemImage,
+    LabeledChoice,
+    NewControlledListItemImageMetadata,
+} from "@/types/ControlledListManager";
+
+const { labeledChoices, image } = defineProps<{
+    labeledChoices: LabeledChoice[];
+    image: ControlledListItemImage;
+}>();
+const item = inject(itemKey) as Ref<ControlledListItem>;
+
+const { $gettext } = useGettext();
+const addMetadata = $gettext("Add metadata");
+
+const newMetadata: Ref<NewControlledListItemImageMetadata> = computed(() => {
+    const otherNewMetadataIds = image.metadata.filter(
+        (metadatum) => typeof metadatum.id === "number"
+    ).map(metadatum => metadatum.id as unknown as number);
```

Comment:
Do you mind sharing the reason why? I haven't done that in other places with multiple constants. Because this one is fatter?

Reply:
Yep! Always happy to defend a decision 😄 

Largely it's to improve readability, and yep it's because the consts are multi-line blocks

[Airbnb styleguide cautions to add newlines between blocks](https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--after-blocks)